mysql-test/suite/innodb/t/innodb_row_lock_time_ms.test

```diff
@@ -19,18 +19,18 @@ SELECT * FROM t FOR UPDATE;
 --connection default
 COMMIT;
 
-SELECT variable_value > 100 FROM information_schema.global_status
+SELECT CAST(variable_value AS INT) > 100 FROM information_schema.global_status
   WHERE LOWER(variable_name) = 'innodb_row_lock_time';
```

Comment:
To obtain more information from the failures, I think that we should follow the example of `storage/rocksdb/mysql-test/rocksdb/t/2pc_group_commit.test`, with something like this:
```sql
SELECT IF(CAST(variable_value AS INT) > 100, 'OK', variable_value) AS innodb_row_lock_time
FROM information_schema.global_status WHERE variable_name='Innodb_row_lock_time';
```